cmd/orb_test.go

```diff
@@ -1070,24 +1070,78 @@ var _ = Describe("Orb integration tests", func() {
 
 		})
 
-		Describe("when listing all orbs with --uncertified", func() {
+		Describe("when listing all orbs with the --json flag", func() {
```

Comment:
This is git diff-ing at an unusual location. The original `Describe` block and test aren't actually deleted or changed. They're further down in this diff as an "added" test.